proposals/compatibility-mode.md

```diff
@@ -392,6 +392,12 @@ If `createTexture()` is called with any of these formats and `sampleCount > 1`,
 
 **Justification**: OpenGL ES 3.1 does not require multisampling to be supported on integer texture formats. The minimum value for GL_MAX_INTEGER_SAMPLES is 1, and [94.7% of reports on gpuinfo have that value](https://opengles.gpuinfo.org/displaycapability.php?name=GL_MAX_INTEGER_SAMPLES&esversion=31).
 
+## 24. Support for `"texture-compression-bc"` may not imply support for `"texture-compression-bc-sliced-3d"` feature.
```

Comment:
Stephen also suggested that we could alternatively just take this requirement out of Core. We only put it there because we could. It's not that important.